dbms/src/Flash/tests/gtest_executors_with_dm.cpp

```diff
@@ -64,11 +64,6 @@ class ExecutorsWithDMTestRunner : public DB::tests::ExecutorTest
                                    toNullableVec<String>("col9", col_string)});
     }
 
-    void TearDown() override
```

Comment:
Why remove this?

Reply:
Because `TaskScheduler::instance.reset()` in `ExecutorTest::TearDown()` will be override.
And `context.mockStorage()->clear();` has been moved to `ExecutorTest::TearDown()`
https://github.com/pingcap/tiflash/pull/6520/files#diff-b4081bb5dbbcbd37da4bb4c8615cf229f7f9dee337158c858856fd744b23ff51R66-R73